v3/src/components/data-summary/data-summary.scss

```diff
@@ -6,6 +6,8 @@
   height: calc(100% - vars.$title-bar-height);
   background-color: aliceblue;
   padding: 5px;
+  border-bottom-left-radius: 6px;
+  border-bottom-right-radius: 6px;
```

Comment:
Since these constants occur in multiple places, perhaps they should be constants in `vars.scss`.